themes/finna2/templates/Helpers/R2RestrictedRecordPrompt.phtml

```diff
@@ -0,0 +1,20 @@
+<div class="r2-restricted-record-register alert alert-info">
+  <div class="r2-restricted-info">
+    <div class="icon"><i class="fa fa-research fa-2x"></i></div>
+  </div>
+  <div class="r2-restricted-content">
+    <?php if ($this->warning): ?>
+      <div><?= $this->transEsc($this->warning) ?></div>
+    <?php endif ?>
+    <?php if ($this->note): ?>
+      <?php // intentionally not escaped so that markup can be used ?>
+      <div><?= $this->translate($this->note) ?></div>
+    <?php endif ?>
+    <?php if ($this->showInfo): ?><div class="metadata"><?= $this->transEsc('R2_restricted_metadata') ?></div><?php endif ?>
+    <?= $this->translate($this->instructions ?? 'R2_restricted_register_instructions_html') ?>
+    <div class="register" role="alert">
+      <p class="register-btn"><a href="<?= $registerFormUrl ?>" data-lightbox data-lightbox-title="<?=$this->transEsc("R2_register_form_title")?>" class="btn btn-primary btn-uppercase btn-r2-register"><i class="fa fa-lg fa-user"></i> <?=$this->transEsc('R2_register')?></a></p>
```

Comment:
href:lle escapeHtmlAttr. Käytä $this->registerFormUrl. Jaa <p> useammalle riville.